nexus/tests/integration_tests/endpoints.rs

```diff
@@ -622,496 +714,751 @@ lazy_static! {
+                    visibility: Visibility::Protected, // XXX is this right?
+                },
             ],
         },
 
         /* Silo identity providers */
 
-        /*
         VerifyEndpoint {
-            url: &*IDENTITY_PROVIDERS_URL, // in ignore list
-            visibility: Visibility::Public,
+            url: &*IDENTITY_PROVIDERS_URL,
             allowed_methods: vec![
-                AllowedMethod::Get,
+                AllowedMethod {
+                    method: MethodAndBody::Get,
+                    visibility: Visibility::PublicNoPermissionRequired,
+                },
             ],
         },
-        */
+
         VerifyEndpoint {
             url: &*SAML_IDENTITY_PROVIDERS_URL,
-            visibility: Visibility::Public,
-            allowed_methods: vec![AllowedMethod::Post(
-                serde_json::to_value(&*SAML_IDENTITY_PROVIDER).unwrap(),
-            )],
+            allowed_methods: vec![
+                AllowedMethod {
+                    method: MethodAndBody::Post(
+                        serde_json::to_value(&*SAML_IDENTITY_PROVIDER).unwrap(),
+                    ),
+                    visibility: Visibility::PublicPermissionRequired,
+                },
+            ],
         },
         VerifyEndpoint {
             url: &*SPECIFIC_SAML_IDENTITY_PROVIDER_URL,
-            visibility: Visibility::Protected,
-            allowed_methods: vec![AllowedMethod::Get],
+            allowed_methods: vec![
+                AllowedMethod {
+                    method: MethodAndBody::Get,
+                    visibility: Visibility::Protected,
+                },
+            ],
+        },
+
+        /* Misc */
+
+        VerifyEndpoint {
+            url: "/session/me",
+            allowed_methods: vec![
+                AllowedMethod {
+                    method: MethodAndBody::Get,
+                    visibility: Visibility::PublicNoPermissionRequired,
+                },
+            ],
+        },
+
+        /* SSH keys */
+
+        VerifyEndpoint {
+            url: &*DEMO_SSHKEYS_URL,
+            allowed_methods: vec![
+                AllowedMethod {
+                    method: MethodAndBody::Get,
+                    visibility: Visibility::Protected, // XXX is this right?
+                },
+                AllowedMethod {
+                    method: MethodAndBody::Post(
+                        serde_json::to_value(&*DEMO_SSHKEY_CREATE).unwrap(),
+                    ),
+                    visibility: Visibility::Protected, // XXX is this right?
+                },
+            ],
+        },
+        VerifyEndpoint {
+            url: &*DEMO_SPECIFIC_SSHKEY_URL,
+            allowed_methods: vec![
+                AllowedMethod {
+                    method: MethodAndBody::Get,
+                    visibility: Visibility::Protected, // XXX is this right?
+                },
+                AllowedMethod {
+                    method: MethodAndBody::Delete,
```

Comment:
How does this not cause us to actually delete the ssh key when we run this test?

Reply:
It would! Any combination of `UnprivilegedAccess::Full` and a DELETE call would. But I'm deferring the SSH keys changes because I couldn't get the authz policy to work and didn't want to hold the global image changes up.